dt: 0.00005
R_rr: 0.33
R_rl: 0.33
tw: 1.57